### DISCUSSION TITLE
杰倫智能科技

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#607] [General]
無程式碼的虛擬資料科學家平台，聽起來就很厲害吧!
https://tw.news.yahoo.com/profet-ai-%E6%9D%B0%E5%80%AB%E6%99%BA%E8%83%BD%E7%A7%91%E6%8A%80%E5%AE%8C%E6%88%90a-%E8%BC%AA%E5%8B%9F%E8%B3%87-062352315.html

**Comment by manipulateed** [comment]
這項科技的確能帶給產業規模的快速提升，並且讓非該專業的人也能嘗試，但這會造成原本這個職業的薪水降低，而不能體現真正專業的人的優勢。
